Loggie/Classes/LogList/LogListTableView.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LogListTableView.swift
+//  Loggie
+//
+//  Created by Damjan Dimovski on 14.10.22.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public struct LogListTableView: UIViewControllerRepresentable {
```

Comment:
```suggestion
public struct LoggieView: UIViewControllerRepresentable {
```

Reply:
I agree the naming is a bit weird and non-SwiftUI like, but I tried to keep it consistent with the rest of the library. If you see all the UIKit stuff, there is no view/view controller with "Loggie" in the name, they all start with "Log", that's why I kept that naming scheme.

Reply:
There is `LoggieManager` which is main entry point to the library.

It still seems to me that it is better to rename it, but I don't have a strong opinion about it.